python/cuml/CMakeLists.txt

```diff
@@ -112,3 +112,6 @@ add_subdirectory(cuml/svm)
 add_subdirectory(cuml/tsa)
 
 add_subdirectory(cuml/experimental/linear_model)
+
+# Include the pth file in the built wheel
+install(FILES "_cuml_accel.pth" DESTINATION ".")
```

Comment:
No idea if this is the proper way to do this, but it seems to work for wheels at least.

I haven't checked if `_cuml_accel.pth` is included in an sdist yet.